proto/buf/alpha/registry/v1alpha1/webhook.proto

```diff
@@ -81,16 +85,25 @@ message ListWebhooksRequest {
 // subscribed webhooks for a given repository.
 message ListWebhooksResponse {
   // The list of subscribed webhooks for a given repository.
-  repeated SubscribedWebhook subscribed_webhooks = 1;
+  repeated Webhook webhooks = 1;
   // The next page token for paginating.
   string next_page_token = 2;
 }
 
-// SubscribedWebhook is the representation of a webhook event and webhook
-// subscription id.
-message SubscribedWebhook {
+// Webhook is the representation of a webhook repository event subscription.
+message Webhook {
   // The event associated with the subscription id.
-  WebhookEvent webhook_event = 1;
+  WebhookEvent event = 1;
   // The id of the associated subscription.
-  string webhook_subscription_id = 2;
+  string webhook_id = 2;
+  // The webhook creation timestamp, immutable.
+  google.protobuf.Timestamp create_time = 3;
+  // The webhook last updated timestamp, mutable.
+  google.protobuf.Timestamp update_time = 4;
```

Comment:
It feels that the mutability/immutability concept isn't really relevant in this API context